src/qml/TrackerSettings.qml

```diff
@@ -0,0 +1,546 @@
+          horizontalAlignment: TextInput.AlignHCenter
+          Layout.preferredWidth: 60
+          Layout.preferredHeight: font.height + 20
+
+          inputMethodHints: Qt.ImhFormattedNumbersOnly
+          validator: DoubleValidator { locale: 'C' }
+
+          onTextChanged: {
+            if( text.length === 0 || isNaN(text) ) {
+              positioningSettings.trackerMinimumDistance = NaN
+            } else {
+              positioningSettings.trackerMinimumDistance = parseFloat( text )
+            }
+          }
+        }
+
+        Label {
+          text: qsTr("Activate sensor constraint")
+          font: Theme.defaultFont
+          wrapMode: Text.WordWrap
+          Layout.fillWidth: true
+
+          MouseArea {
+            anchors.fill: parent
+            onClicked: sensorCapture.toggle()
+          }
+        }
+
+        QfSwitch {
+          id: sensorCapture
+          Layout.preferredWidth: implicitContentWidth
+          Layout.alignment: Qt.AlignTop
+          checked: false
+          onCheckedChanged: {
+            positioningSettings.trackerSensorCaptureConstraint = checked
+          }
+        }
+
+        Label {
+          text: qsTr("Record when all active constraints are met")
+          font: Theme.defaultFont
+          wrapMode: Text.WordWrap
+          Layout.fillWidth: true
+          enabled: (timeInterval.checked + minimumDistance.checked + sensorCapture.checked) > 1
+          visible: (timeInterval.checked + minimumDistance.checked + sensorCapture.checked) > 1
+
+          MouseArea {
+            anchors.fill: parent
+            onClicked: allConstraints.toggle()
+          }
+        }
+
+        QfSwitch {
+          id: allConstraints
+          Layout.preferredWidth: implicitContentWidth
+          Layout.alignment: Qt.AlignTop
+          enabled: (timeInterval.checked + minimumDistance.checked + sensorCapture.checked) > 1
+          visible: (timeInterval.checked + minimumDistance.checked + sensorCapture.checked) > 1
+          checked: false
+          onCheckedChanged: {
+            positioningSettings.trackerMeetAllConstraints = checked
+          }
+        }
+
+        Label {
+          text: qsTr( "When enabled, vertices with only be recorded when all active constraints are met. If the setting is disabled, individual constraints met will trigger a vertex addition." )
+          font: Theme.tipFont
+          color: Theme.secondaryTextColor
+          textFormat: Qt.RichText
+          wrapMode: Text.WordWrap
+          Layout.fillWidth: true
+          enabled: (timeInterval.checked + minimumDistance.checked + sensorCapture.checked) > 1
+          visible: (timeInterval.checked + minimumDistance.checked + sensorCapture.checked) > 1
+        }
+
+
+        Label {
+          text: sensorCapture.checked
+                ? qsTr( "When the sensor constraint is activated alone, vertex additions will occur whenever sensor has captured new data." )
```

Comment:
```suggestion
                ? qsTr( "When the sensor constraint is activated alone, vertex additions will occur whenever the sensor has captured new data." )
```
When reading I feel like there is a missing article "the", but my English is not perfect.